docs/building-images.md

```diff
@@ -128,6 +134,11 @@ operatingSystem:
   * `encryptedPassword` - Optional; Encrypted password to set for the use (for example, using `openssl passwd -6 $PASSWORD`
     to generate the value for this field).
   * `sshKeys` - Optional; List of public SSH keys to configure for the user.
+  * `primaryGroup` - Optional; If specified, the user will be configured with this as the primary group. The group
```

Comment:
If it is not specified, what would be the behavior? I will expect a group with the same name as the user would be created (same as `useradd` would do if not specified) but I guess it would be nice to explicitly document it.

Reply:
It ends up as the default for what the OS does, which is `users` in SLE Micro's case. But good call, I should explicitly mention that.